pkg/virt-controller/services/rendercontainer_test.go

```diff
@@ -243,6 +245,40 @@ var _ = Describe("Container spec renderer", func() {
 				Expect(specRenderer.Render(exampleCommand).LivenessProbe).To(Equal(probeWithDelay(probe)))
 			})
 		})
+
+		Context("liveness exec probe", func() {
+			var expectedExecCmd = []string{
+				"virt-probe",
+				"--domainName", "_",
+				"--timeoutSeconds", strconv.FormatInt(int64(dummyProbe().TimeoutSeconds), 10),
+				"--command", "dummy-cli",
+				"--",
+			}
+
+			It("its pod should feature the same exec probe but wrapped in a virt-probe command", func() {
+				probe := dummyProbe()
+				probe.Handler = v1.Handler{
+					Exec: &k8sv1.ExecAction{Command: []string{"dummy-cli"}},
+				}
+				specRenderer = NewContainerSpecRenderer(containerName, img, pullPolicy, WithLivelinessProbe(
+					vmiWithLivenessProbe(probe)))
+				Expect(specRenderer.Render(exampleCommand).LivenessProbe.Exec.Command).To(Equal(expectedExecCmd))
```

Comment:
You can get rid of `expectedExecCmd` context scope variable with:
```golang
				Expect(specRenderer.Render(exampleCommand).LivenessProbe.Exec.Command).To(HaveExactElements(
					"virt-probe",
					"--domainName", "_",
					"--timeoutSeconds", strconv.FormatInt(int64(dummyProbe().TimeoutSeconds), 10),
					"--command", "dummy-cli",
					"--"))
```
OR
please move the `expectedExecCmd` inside the `It` block.
Even if currently there is only one `It`, this could not be true forever in the future, and this global var can lead to errors.

Reply:
fixed as requested